common/tier4_autoware_utils/include/tier4_autoware_utils/ros/polling_reciever.hpp

```diff
@@ -0,0 +1,55 @@
+// Copyright 2024 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef TIER4_AUTOWARE_UTILS__ROS__POLLING_RECEIVER_HPP_
+#define TIER4_AUTOWARE_UTILS__ROS__POLLING_RECEIVER_HPP_
+
+#include <rclcpp/rclcpp.hpp>
+
+namespace tier4_autoware_utils
+{
+
+template <typename T>
+class InterProcessPollingReceiver
```

Comment:
`InterProcessPollingReceiver` is hard to understand for me.
`InterProcessSubscriber` sounds easy to understand.

@takam5f2 @ohsawa1204 What do you think?

Reply:
> `InterProcessPollingReceiver` is hard to understand for me. `InterProcessSubscriber` sounds easy to understand.
> 
> @takam5f2 @ohsawa1204 What do you think?

@takayuki5168 
I agree!

Reply:
OK, I see.
However, `InterProcess` denotes less meaning in this context, hence we need some additional modifier term.
How do you think?

Reply:
To recognize the difference to between the conventional call back subscriber, we use `InterProcessPollingSubscriber` as the name of this class